index.js

```diff
@@ -1,202 +1,15 @@
 const {authContext, appContext, pairedDeviceContext, testContext} = require('./lib/context');
 
 const {connectToIpcAndBootstrapSession} = require('./lib/utils/sessionStarter');
-const {warnUnusedLeaves} = require('./lib/utils/unusedExpressionWatchers');
-const {warnLauncherAndLibHasDiffVersions} = require('./lib/utils/packageMetadataHelper');
-
-// Publicly available API goes here
-class SUITEST_API {
-	constructor() {
-		this.openSession = openSession;
-		this.closeSession = closeSession;
-		this.startTestPack = startTestPack;
-		this.startTest = startTest;
-		this.endTest = endTest;
-		this.pairDevice = pairDevice;
-		this.releaseDevice = releaseDevice;
-		this.setAppConfig = setAppConfig;
-		this.configure = configure;
-		this.interactive = interactive;
-
-		this.openApp = openApp;
-		this.openUrl = openUrl;
-		this.application = application;
-		this.clearAppData = clearAppData;
-		this.cookie = cookie;
-		this.location = location;
-		this.sleep = sleep;
-		this.press = pressButton;
-		this.position = position;
-		this.window = window;
-		this.executeCommand = executeCommand;
-		// this.executeBrightScript = executeBrightScript;
-		this.jsExpression = jsExpression;
-		// this.brightScriptExpression = brightScriptExpression;
-		this.element = element;
-		this.video = video;
-		this.psVideo = playstationVideo;
-		this.networkRequest = networkRequest;
-		this.pollUrl = pollUrl;
-
-		this.PROP = ELEMENT_PROP;
-		this.COMP = PROP_COMPARATOR;
-		this.VALUE = VALUE;
-		this.VIDEO_STATE = VIDEO_STATE;
-		this.VISIBILITY_STATE = VISIBILITY_STATE;
-		this.CONTENT_MODE = CONTENT_MODE;
-		this.ELEMENT_STATE = ELEMENT_STATE;
-		this.TEXT_ALIGNMENT = TEXT_ALIGNMENT;
-		this.IMAGE_LOAD_STATE = IMAGE_LOAD_STATE;
-		this.BORDER_STYLE = BORDER_STYLE;
-		this.VRC = VRC;
-		this.KEY = KEY;
-		this.NETWORK_PROP = NETWORK_PROP;
-		this.NETWORK_METHOD = NETWORK_METHOD;
-		this.HAD_NO_ERROR = HAD_NO_ERROR;
-
-		this.authContext = authContext;
-		this.appContext = appContext;
-		this.pairedDeviceContext = pairedDeviceContext;
-		this.testContext = testContext;
-
-		// For testing
-		this.webSockets = webSockets;
-
-		this.assert = {
-			application: applicationAssert,
-			clearAppData: clearAppDataAssert,
-			openApp: openAppAssert,
-			openUrl: openUrlAssert,
-			location: locationAssert,
-			sleep: sleepAssert,
-			press: pressButtonAssert,
-			position: positionAssert,
-			window: windowAssert,
-			executeCommand: executeCommandAssert,
-			// executeBrightScript: executeBrightScriptAssert,
-			jsExpression: jsExpressionAssert,
-			// brightScriptExpression: brightScriptExpressionAssert,
-			cookie: cookieAssert,
-			element: elementAssert,
-			pollUrl: pollUrlAssert,
-			runTest: runTestAssert,
-			networkRequest: networkRequestAssert,
-			video: videoAssert,
-			psVideo: playstationVideoAssert,
-		};
-	}
-}
+const SUITEST_API = require('./suitest');
+const suitest = new SUITEST_API(
+	true, {authContext, appContext, pairedDeviceContext, testContext}
```

Comment:
I would transform first argument to some settings object, for ability adding new options without changing constructor signature.